x-pack/plugins/security_solution/common/field_maps/field_names.ts

```diff
@@ -17,8 +17,23 @@ export const ALERT_THRESHOLD_RESULT = `${ALERT_NAMESPACE}.threshold_result` as c
 export const ALERT_THRESHOLD_RESULT_COUNT = `${ALERT_THRESHOLD_RESULT}.count` as const;
 export const ALERT_NEW_TERMS = `${ALERT_NAMESPACE}.new_terms` as const;
 export const ALERT_NEW_TERMS_FIELDS = `${ALERT_RULE_PARAMETERS}.new_terms_fields` as const;
-export const ALERT_HOST_CRITICALITY = `${ALERT_NAMESPACE}.host.criticality_level` as const;
-export const ALERT_USER_CRITICALITY = `${ALERT_NAMESPACE}.user.criticality_level` as const;
+/**
+ * @deprecated Use ALERT_HOST_CRITICALITY
+ */
+export const LEGACY_ALERT_HOST_CRITICALITY = `${ALERT_NAMESPACE}.host.criticality_level` as const;
+/**
+ * @deprecated Use ALERT_USER_CRITICALITY
+ */
+export const LEGACY_ALERT_USER_CRITICALITY = `${ALERT_NAMESPACE}.user.criticality_level` as const;
+
+export const ALERT_HOST_CRITICALITY = `host.asset.criticality` as const;
+export const ALERT_USER_CRITICALITY = `user.asset.criticality` as const;
+export const ALERT_HOST_RISK_SCORE_CALCULATED_LEVEL = `host.risk.calculated_level` as const;
```

Comment:
Do these need these `ALERT_` prefixes if the fields aren't specific to alerts?

Reply:
Yes, for those fields I want to show that they are related to alerts.

We do have `host.risk.calculated_level` in other places in our app, but `host.asset.criticality` unique only for alerts, so I wanted to specify that